src/main/java/org/spongepowered/common/config/serializer/DataSerializableTypeSerializer.java

```diff
@@ -22,7 +22,7 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package org.spongepowered.common.config;
+package org.spongepowered.common.config.serializer;
```

Comment:
Since there were now 4 TypeSerializers in the config package, I thought it would be better to put them in their own package, to not clutter the rest of the contents of the config package